### ISSUE TITLE
Describe accepted return types/classes of `f` in `epi[x]_slide` roxygen

### ISSUE THREAD

**Issue by brookslogan** [cmu-delphi/epiprocess#234]
There are constraints on the functions `f` that we accept in the slide functions. We need to describe this in the docs.

The code is hard to parse and hard to say what is actually accepted in `f`.

**Comment by dshemetov** [comment]
@nmdefries @brookslogan any suggestions for how to tackle this?

**Comment by brookslogan** [comment]
I'm guessing this will involve some amount of rooting through source code and testing.  And some may be in flux with `epi_slide()` work.

Check:
- For tidyeval way: can we output vectors? tibbles? lists of tibbles or other things?
- For non-tidyeval way: same questions
- What lengths/nrows can we output? (epi_slide has more demands here; either 1 or the number it's thinking of; epix_slide shouldn't)
